android/tangram/src/com/mapzen/tangram/MapController.java

```diff
@@ -88,6 +88,8 @@ public MapController(Activity mainApp, MapView view, String sceneFilePath) {
         view.setRenderer(this);
         view.setRenderMode(GLSurfaceView.RENDERMODE_WHEN_DIRTY);
 
+        init(this, assetManager, scenePath);
```

Comment:
Actually can you put a comment here explaining init call being called twice? We had a discussion on this, but good to have a comment in the code.